### PR TITLE
Add config for audit trail monolog logger

### PR DESCRIPTION
**Pull request by bjorvack**
zie https://app.activecollab.com/108877/reports/team-timeline?modal=Task-194357-533

### CODE REVIEW

File: src/Skeleton/PostCreateProject.php [hunk 1]
```diff
@@ -376,6 +376,44 @@ private static function reconfigureApplication(Event $event): void
...
+        preg_match('|monolog:|', $content, $matches, PREG_OFFSET_CAPTURE);
+        $offset = mb_strpos($content, PHP_EOL, $matches[0][1]) + 1;
+        $insert = [
+            '   handlers:',
+            '       audit_trail:',
+            '           type: rotating_file',
+            '           path: "%kernel.logs_dir%/audit.log"',
+            '           level: info',
+            '           channels: [\'audit_trail\']',
+            '           max_files: 10',
```
Comment (jonasdekeukelaere) [inline src/Skeleton/PostCreateProject.php:393]: 10 files is 10 dagen?

Comment (bjorvack) [reply src/Skeleton/PostCreateProject.php:393]: is indd 10 dagen

Comment (tijsverkoyen) [reply src/Skeleton/PostCreateProject.php:393]: Doe maar 30 dagen by default.

Comment (bjorvack) [reply src/Skeleton/PostCreateProject.php]: fixed aa9bbec86c3730f2228d2b6973dd5d9c72ee5b75

**Review by tijsverkoyen — APPROVED** [review]

**Pull request closed without merging**